components/layout_thread_2020/lib.rs

```diff
@@ -553,13 +563,15 @@ impl LayoutThread {
     fn build_layout_context<'a>(
         &'a self,
         guards: StylesheetGuards<'a>,
+        script_initiated_layout: bool,
```

Comment:
Maybe an enum, `LayoutRequestOrigin` or something?

Reply:
This is copied from `components/layout_thread`